src/lib/skilling/skills/mining.ts

```diff
@@ -250,6 +259,20 @@ const ores: Ore[] = [
 	}
 ];
 
+const MotherlodeMines: MotherlodeMine[] = [
```

Comment:
This makes no sense to me. Why is it an array?

Reply:
I think it's just a result of that we had 2 motherlode mine options in past, upper and lower which then got dismissed. but yes the [ ] should be removed.